package.json

```diff
@@ -20,14 +21,8 @@
     "type": "git",
     "url": "git+https://github.com/NodeSecure/rc.git"
   },
-  "files": [
-    "dist"
-  ],
-  "keywords": [
-    "rc",
-    "config",
-    "configuration"
-  ],
+  "files": ["dist"],
+  "keywords": ["rc", "config", "configuration"],
```

Comment:
Did you execute prettier or something here?

Reply:
Yes indeed, i fixed it in this commit : 50c4e0f